features/handled_errors.feature

```diff
@@ -22,7 +22,7 @@ Feature: Reporting handled errors
       | ios     | iOS     |
       | macos   | macOS   |
     And the event "app.version" equals "1.0"
-    And the event "device.freeDisk" is not null
+    And unless iOS|tvOS|watchOS, the event "device.freeDisk" is not null
```

Comment:
Might be good to involve @twometresteve here for an opinion, but I'd suggest that we stick to only the Cocoa platforms we support for Unreal (iOS and macOS), in which case you just need the negation of the `/^on (Android|iOS), (.+)/` step below to do "unless on iOS"?

Reply:
Yeah, I agree - I can see how you arrived at the list, but I would keep it simpler by removing tvOS and watchOS from the Unreal scope.  We're not professing support for them (if they are even possible) or testing with on either.

Reply:
I wanted to put in exceptions specifically for the platforms we *don't* support a particular feature on (regardless of the actual support we offer for the platform), so that if in future we do add more platform support, we don't have to go back and fix up all the tests to add the new required positive matches. With a negative match for each specific unsupported feature, we can just set it and forget it.

Reply:
🤔 .... for this particular case I can see where you're coming from when iOS is one of these excluded.  If it was just tvOS/watchOS being excluded, or we had lots of conditions like this, then I would make a case for still changing the wording based on YAGNI and it being visual noise - but now I think it's fine as it is.  Fine either way really.